keras/utils/np_utils.py

```diff
@@ -91,3 +91,47 @@ def normalize(x, axis=-1, order=2):
     l2 = np.atleast_1d(np.linalg.norm(x, order, axis))
     l2[l2 == 0] = 1
     return x / np.expand_dims(l2, axis)
+
+
+@keras_export("keras.utils.to_ordinal")
+def to_ordinal(y, num_classes=None, dtype="float32"):
+    """Converts a class vector (integers) to an ordinal class matrix for ordinal
+        regression/classification
+
+    Args:
+        y: Array-like with class values to be converted into a matrix
+            (integers from 0 to `num_classes - 1`).
+        num_classes: Total number of classes. If `None`, this would be inferred
+          as `max(y) + 1`.
```

Comment:
Please use 4 spaces as the indent.

Reply:
I think you meant the 2 spaces before `as max(y) + 1` which I converted to 4 spaces in the following commit. If so I think `to_categorical` also uses 2 spaces from which I adopted the format.

Also noticed after `black` formatting it creates a `line too long` error in `fake8` as mentioned in psf/black/issues/1161

Reply:
> Please also update [this file](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) after the PR is merged, by adding to the tf.keras section for new features.

@haifeng-jin should I create a PR in `tensorflow` for that?

Reply:
The spaces in docstrings are not consistent through the codebase. We try to gradually change them to use 4 spaces. 

In the conflict of black and flake8, we need to manually resolve them.

Yes, you should create a PR in tensorflow afterwards.
Thank you!